packages/augur-core/source/contracts/BaseSimpleDex.sol

```diff
@@ -0,0 +1,182 @@
+pragma solidity 0.5.15;
+
+import "ROOT/IAugur.sol";
+import "ROOT/ISimpleDex.sol";
+import "ROOT/CashSender.sol";
+import "ROOT/libraries/token/IERC20.sol";
+import "ROOT/libraries/token/VariableSupplyToken.sol";
+import "ROOT/libraries/Initializable.sol";
+import "ROOT/libraries/ReentrancyGuard.sol";
+import "ROOT/libraries/math/SafeMathUint256.sol";
+
+
+contract BaseSimpleDex is Initializable, ReentrancyGuard, VariableSupplyToken, CashSender {
+    using SafeMathUint256  for uint256;
+
+    string constant public name = "Simple Decentralized Exchange";
+    string constant public symbol = "SDE";
+
+    IAugur public augur;
+    address public token;
+    IERC20 public cash;
```

Comment:
Consider removing this variable.  We should never be referencing it directly (always through `CashSender`) and as seen in other comments on this PR its presence tempts one to actually use it.